packages/sources/layer2-sequencer-health/README.md

```diff
@@ -39,13 +41,17 @@ For the adapter to be useful on the desired network, at least one endpoint (RPC
 
 ### Sample Output
 
+0 = Sequencer is healthy
+1 = Sequencer is unhealthy
+
 ```json
 {
-  "jobRunID": "278c97ffadb54a5bbb93cfec5f7b5503",
+  "jobRunID": "1",
+  "result": 0,
+  "statusCode": 200,
   "data": {
-    "result": true,
-    "isHealthy": true
-  },
-  "statusCode": 200
+    "isHealthy": 0,
+    "result": 0
+  }
```

Comment:
This doesn't sit right with me:
`"isHealthy": 1 == sequencer is unhealthy`

Can we improve this? Change to `isUnhealthy`?

Reply:
What about if we just return the Boolean value for isHealthy? I don’t think we can switch the 0/1 value for the result field as we already have feeds that are dependent on 0 representing a healthy sequencer and 1 representing an unhealthy sequencer.

Reply:
Yeah that could work as well. I was not suggesting that we change the 0/1 value, but rather rename the return param `isHealthy` to `isUnhealthy`. I'm guessing we're only using the `result` param, so renaming this additional param wouldn't have any effect?

Reply:
Gotcha. Yeah think that we’re only using the result param so this should not have any effect.